plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -265,6 +269,49 @@ public void testCharVarcharComparison()
                 .hasStackTraceContaining("Unsupported type: char(3)");
     }
 
+    @Test(dataProvider = "timestampValues")
+    public void testTimestampPredicatePushdown(String value)
+    {
+        String tableName = "test_parquet_timestamp_predicate_pushdown_" + randomTableSuffix();
+
+        assertUpdate("DROP TABLE IF EXISTS " + tableName);
+        assertUpdate("CREATE TABLE " + tableName + " (t TIMESTAMP WITH TIME ZONE)");
+        assertUpdate("INSERT INTO " + tableName + " VALUES (TIMESTAMP '" + value + "')", 1);
+
+        DistributedQueryRunner queryRunner = (DistributedQueryRunner) getQueryRunner();
+        ResultWithQueryId<MaterializedResult> queryResult = queryRunner.executeWithQueryId(
+                getSession(),
+                "SELECT * FROM " + tableName + " WHERE t < TIMESTAMP '" + value + "'");
+        assertEquals(getQueryInfo(queryRunner, queryResult).getQueryStats().getProcessedInputDataSize().toBytes(), 0);
+
+        queryResult = queryRunner.executeWithQueryId(
+                getSession(),
+                "SELECT * FROM " + tableName + " WHERE t > TIMESTAMP '" + value + "'");
+        assertEquals(getQueryInfo(queryRunner, queryResult).getQueryStats().getProcessedInputDataSize().toBytes(), 0);
+
+        assertQueryStats(
+                getSession(),
+                "SELECT * FROM " + tableName + " WHERE t = TIMESTAMP '" + value + "'",
+                queryStats -> assertThat(queryStats.getProcessedInputDataSize().toBytes()).isGreaterThan(0),
+                results -> {});
+    }
+
+    @DataProvider
+    public Object[][] timestampValues()
+    {
+        return new Object[][] {
+                {"1965-10-31 01:00:08.123 UTC"},
```

Comment:
Please add entries to test handling of gaps and ambiguous values in the JVM time zone. Note that these may appear as partition values as well.

Reply:
@MiguelWeezardo Added a below entry. Please take another look.
```
1970-01-01 01:13:42.000 America/Bahia_Banderas
```